survos/command-bundle/1.5/config/routes/survos_command.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// config/routes.php
+use Survos\CommandBundle\Controller\CommandController;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return function (RoutingConfigurator $routes) {
+    $routes->add('survos_commands', '/admin/commands')
+        ->controller([CommandController::class, 'commands'])
+    ;
+
+    $routes->add('survos_command', 'admin/run-command/{commandName}')
+        ->controller([CommandController::class, 'runCommand'])
+    ;
```

Comment:
It's pretty common to just have 1 route "import from inside the user's code, which then points to another file that adds the individual routes.

Reply:
I've made that change now, I think it follows the standard conventions.